runtimes/runtimes/standalone.ts

```diff
@@ -183,6 +194,12 @@ export const standalone = (props: RuntimeProps) => {
             },
         }
 
+        const notifications: Notifications = {
+            showNotifications: params => lspConnection.sendNotification(showNotificationsRequestType.method, params),
```

Comment:
Given we are defining, `showNotificationsRequestType`, wouldn't it work to do: `lspConnection.sendNotification(showNotificationsRequestType, params)`

Not sure if it changes things, but I vaguely remember that it could sometimes make a difference

Reply:
I ran into a problem attempting to use the form like `lspConnection.sendNotification(showNotificationsRequestType, params)` in the vscode sample client.  I had to use the `.method` form.

There are more details in one of my recent PRs:  https://github.com/aws/language-servers/pull/489#discussion_r1795690421